backstory/src/main/webapp/vision-demo.js

```diff
@@ -137,10 +139,51 @@ function formatLabelsAsTable(labels) {
     scoreData.innerText = `${(label.score * 100).toFixed(2)}%`;
     scoreData.classList.add('labels-table-cell');
   }
-  
+
   return table;
 }
 
+// VALIDATE IMAGE UPLOAD
+
+/**
+ * Validate that the file uploaded to image upload
+ * is an accepted image. Alert user to upload a new file if not.
+ *
+ * @return true, if valid image uploaded; false, if no image uploaded
+ *      or if no valid image uploaded
+ */
+function validateImageUpload() {
```

Comment:
Similar to my comment above, could we export this functionality from the other JS file?